baselines/webworker.generated.d.ts

```diff
@@ -1287,18 +1380,50 @@ interface URL {
     port: string;
     protocol: string;
     search: string;
-    username: string;
     readonly searchParams: URLSearchParams;
+    username: string;
     toString(): string;
 }
 
 declare var URL: {
     prototype: URL;
-    new(url: string, base?: string | URL): URL;
+    new(url: string, base?: string): URL;
```

Comment:
This looks like a regression. We don't allow passing `URL` as a string, since it isn't one, but passing a URL as the second parameter here seems important.

Reply:
will fix it

Reply:
done.